### DISCUSSION TITLE
community/

### DISCUSSION THREAD

**Discussion by giscus** [new-epilepsy/new-epilepsy#7] [General]
# discussion/

Discussion.

http://127.0.0.1:4000/discussion/

**Comment by new-epilepsy** [comment]
### 📣 Hello! 

We’re using GitHub Discussions as a place to connect with other members of NEW. We hope that you:

- Ask questions you’re wondering about.
- Promote your work.
- Share ideas, interesting articles, job ads, talks, etc.
- Engage and collaborate with other NEW members.
- Welcome others and are open-minded.

Don't be shy ✨

**Comment by new-epilepsy** [comment]
#### JOB OPPORTUNITIES 🧰

Are you passionate about advancing the field of computational neurology? [Newcastle University](https://www.ncl.ac.uk/) is seeking talented individuals to join their dynamic team. They have several exciting opportunities currently open and on the horizon! 

Find out more [here »](https://new-epilepsy.com/new/2024.01.10/)
